src/utils.ts

```diff
@@ -319,3 +319,6 @@ export const getFormInitialValues = (fieldNames: object) =>
     (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
     {},
   );
+
+// Time (in milliseconds ) that the message is displayed on the screen.
+export const successMessageDelay = 3000;
```

Comment:
@cbernat should it be uppercase or lowercase?

Reply:
We use it this way, I see it ok.